db/schema.cds

```diff
@@ -7,11 +7,13 @@ entity Books : cuid, managed {
   descr    : localized String(1111);
   author   : Association to Authors;
   stock    : Integer;
+  @assert.range: [0.01, 99999.00]
```

Comment:
One could argue, why the Decimal allows 9 digits, and the validation only allows a total of up to 7 digits.

Reply:
I would actually like to remove this one, as I believe one assertion is enough to showcase the feature in general. Would you agree?

Reply:
Yes, makes sense!